xcc/job.py

```diff
@@ -93,23 +93,32 @@ class Job:
 
     @staticmethod
     def list(
-        connection: Connection, limit: int = 5, ids: Optional[Collection[str]] = None
+        connection: Connection,
+        limit: int = 5,
+        ids: Optional[Collection[str]] = None,
+        status: str = None,
     ) -> Sequence[Job]:
         """Returns jobs submitted to the Xanadu Cloud.
 
+
         Args:
             connection (Connection): connection to the Xanadu Cloud
             limit (int): maximum number of jobs to retrieve
             ids (Collection[str], optional): IDs of the jobs to retrieve; if at
                 least one ID is specified, ``limit`` will be set to the length
                 of the ID collection
+            status (str, optional): status parameter used to filter the jobs list.
+                If status is empty, return all the jobs
```

Comment:
We should reuse the wording from the CLI and be careful about interchanging "empty" (i.e., `""`) with `None`:
```suggestion
            status (str, optional): filter jobs by the given status (if not ``None``)
```